package-lock.json

```diff
@@ -1,6901 +1,8 @@
 {
   "name": "xero-node",
-  "version": "4.11.0",
-  "lockfileVersion": 2,
+  "version": "4.11.3",
```

Comment:
wrong version number in package-lock, doesn't match package.json?

Reply:
Fixed!